eks-distro-base/tests/run_tests.sh

```diff
@@ -163,22 +219,118 @@ function check_base-iptables() {
+            echo "ip6tables wrapper alternative issue!"
+            exit 1
+        fi
+        
+        # iptables-wrapper defaults to nft mode when none of the kubelet hint rules are set
+        if docker run --rm --platform=$platform $LOCAL_REGISTRY/eks-distro-minimal-images-base-test:iptables-wrapper-latest iptables --version | grep -v 'nf_tables'; then
+            echo "iptables wrapper issue!"
+            exit 1
+        fi
+
+        if docker run --rm --platform=$platform $LOCAL_REGISTRY/eks-distro-minimal-images-base-test:iptables-wrapper-latest ip6tables --version | grep -v 'nf_tables'; then
+            echo "ip6tables wrapper issue!"
+            exit 1
+        fi
+        
+        # create KUBE-IPTABLES-HINT and test that wrapper sets the correct mode        
+        for mode in nft legacy; do
+            local expected="$mode"
+            if [[ "$mode" == "nft" ]]; then
+                expected="nf_tables"
+            fi
+
+            # no matter which version of a given iptables mode is used to add the hint, it should trigger the wrapper to use that mode for both
+            for ip in iptables ip6tables; do
+                cleanup-iptable-rules
+
+                docker run --privileged --rm --net host --platform=$platform $LOCAL_REGISTRY/eks-distro-minimal-images-base-test:iptables-$mode-latest $ip -t mangle -N KUBE-IPTABLES-HINT
+
+                if docker run --privileged --net host --rm --platform=$platform $LOCAL_REGISTRY/eks-distro-minimal-images-base-test:iptables-wrapper-latest iptables --version | grep -v "$expected"; then
+                    echo "iptables wrapper issue!"
+                    exit 1
+                fi
+                
+                if docker run --privileged --net host --rm --platform=$platform $LOCAL_REGISTRY/eks-distro-minimal-images-base-test:iptables-wrapper-latest ip6tables --version | grep -v "$expected"; then
+                    echo "iptables wrapper issue!"
+                    exit 1
+                fi
+
+                if ! docker run --privileged --net host --rm --platform=$platform $LOCAL_REGISTRY/eks-distro-minimal-images-base-test:iptables-wrapper-with-bash-latest bash -c "$ip --version && update-alternatives --list" | grep "iptables	manual	/usr/sbin/iptables-$mode"; then
+                    echo "iptables wrapper alternative issue!"
+                    exit 1
+                fi
+
+                if ! docker run --privileged --net host --rm --platform=$platform $LOCAL_REGISTRY/eks-distro-minimal-images-base-test:iptables-wrapper-with-bash-latest bash -c "$ip --version && update-alternatives --list" | grep "ip6tables	manual	/usr/sbin/ip6tables-$mode"; then
+                    echo "ip6tables wrapper alternative issue!"
+                    exit 1
+                fi
+            done
+        done
+        
+        cleanup-iptable-rules
+
+        # run upstream test binaries which tests a number of the same cases as above
```

Comment:
is anything that you are testing above this line missing from the go tests?
If so, let me and I'll add it.